### ISSUE TITLE
arrays with certain element-types can't be created

### ISSUE THREAD

**Issue by dkochmanski** [drmeister/clasp#280]
unsigned-byte arrays may be created up with arbitrary size, but any size above 60 ends with a condition.

Testcase:

``` common-lisp
> (make-array 10 :element-type `(signed-byte 61)
                            :initial-element 0)

Condition of type: SIMPLE-PROGRAM-ERROR
File(../../include/clasp/core/numbers.h) lineNumber(197): Subclass[BIGNUM] must implement method[negate_]

Available restarts:
(use :r1 to invoke restart 1)

1. (RESTART-TOPLEVEL) Go back to Top-Level REPL.

Broken at frame[122] CALL-WITH-VARIABLE-BOUND.
 File: #P"/data/Warsztat/Repozytoria/clasp/src/core/compiler.cc" (Position #37160)
```

Bit array:

``` common-lisp
(make-array 5 :element-type 'bit
                  :displaced-to #*0111000110
                  :displaced-index-offset 3)


Condition of type: SIMPLE-PROGRAM-ERROR
File(../../include/clasp/core/lispVector.h) lineNumber(103): Subclass[SIMPLE-BIT-VECTOR] must implement method[elt]
Available restarts:
(use :r1 to invoke restart 1)

1. (RESTART-TOPLEVEL) Go back to Top-Level REPL.

Broken at frame[109] CALL-WITH-VARIABLE-BOUND.
 File: #P"/data/Warsztat/Repozytoria/clasp/src/core/compiler.cc" (Position #37160)
```

// EDIT: I won't create a separate issue for this one, just change the issue name

**Comment by Shinmera** [comment]
Error implies that this is a dupe of #246

**Comment by Shinmera** [comment]
Confirmed fixed on dcc6f10e04e998f628a90922797eeca6dd8b68f9
